src/saving/SaveListItem.tscn

```diff
@@ -275,8 +276,8 @@ margin_left = 1354.0
 margin_right = 1854.0
 margin_bottom = 128.0
 rect_min_size = Vector2( 500, 0 )
-DialogText = "DELETE_SAVE_CONFIRMATION"
 WindowTitle = "DELETE_THIS_SAVE"
+DialogText = "DELETE_SAVE_CONFIRMATION"
```

Comment:
I guess this reordering doesn't cause save name changes? Though, I think this should probably be reverted, as I think in Godot 3.4 I've now seen things most of the time being properly in alphabetical order...